.gitignore

```diff
@@ -20,4 +20,6 @@ elf-cleaner.sh
 .vscode/
 
 dalvikdx/bin/
-dexmaker/bin/
\ No newline at end of file
+dexmaker/bin/
+edxp-core/.cxx/
+edxp-core/template_override/
```

Comment:
Why ignore template_override?

Reply:
it includes these generated files...
- system/framework/eddalvikdx.jar
- system/framework/eddexmaker.jar
- system/lib/libwhale.edxp.so
- system/lib64/libwhale.edxp.so

I dunno, maybe I should be more specific on .gitignore...

Reply:
okay I just skip this commit... it's not that important for this PR's purpose.